style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #1b1b32;
+  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-green: #37af65;
+}

```

Comment:
Good use of CSS rule for multiple elements

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear
- [ ] However, this as I can see is not really a "problem" that needs to be fixed.